packages/main/src/StepInput.ts

```diff
@@ -512,19 +530,53 @@ class StepInput extends UI5Element implements IFormElement {
 		}
 	}
 
+	_isValueWithCorrectPrecision() {
+		// gets either "." or "," as delimiter which is based on locale, and splits the number by it
+		const delimiter = this.input.value.includes(".") ? "." : ",";
```

Comment:
Can you explain why we can't have scenarios for thousands separated by commas here? And why we only check "," and "."

Reply:
1. We can't have scenarios where thousands are separated by commas here, because we do **not** have formatter or utility at all to format the whole numbers (thousands) with commas unlike in UI5.
2. About why we check only ',' and '.' - these are the only conventionally accepted decimal separators for decimal numbers and we do not support any other ones.